client/js/components/procedure/StatementSegmentsList/StatementSegmentsList.vue

```diff
@@ -539,9 +553,11 @@ export default {
     },
 
     saveStatement (statement) {
+      this.$on('save', this.canUpdateStatementAttributesFullText(statement))
```

Comment:
Why is `this.$on('save' ...)` needed? Isn't `saveStatement` called `@save` anyway?